osfoffline/utils/__init__.py

```diff
@@ -5,18 +5,18 @@
 import logging.config
 import logging.handlers
 import os
-import sys
```

Comment:
sys.excepthook was breaking a lot of things. Uncaught exceptions will be logged anyways